keras/src/backend/tensorflow/numpy.py

```diff
@@ -23,6 +23,19 @@
 from keras.src.backend.tensorflow.core import shape as shape_op
 
 
+def rot90(array, k=1, axes=(0, 1)):
+    """Rotate an array by 90 degrees in the specified plane."""
+    if array.ndim < 2:
+        raise ValueError(
+            f"Input array must have at least 2 dimensions. Received: array.ndim={array.ndim}"
+        )
+    if len(axes) != 2 or axes[0] == axes[1]:
+        raise ValueError(
+            f"Invalid axes: {axes}. Axes must be a tuple of two different dimensions."
+        )
+    return tf.experimental.numpy.rot90(array, k=k, axes=axes)
```

Comment:
Is it possible to use something that's not `tf.experimental.numpy`? It has proven fairly unreliable in the past (often hardcoding an eager assumption or not being XLA compatible)